### DISCUSSION TITLE
Invalidating server-side procedure calls?

### DISCUSSION THREAD

**Discussion by MoritzKronberger** [icflorescu/trpc-sveltekit#36] [Q&A]
Since trpc-sveltekit allows procedure calls inside the `load` function, by passing `load`-`fetch` to the client, it would be nice to be able to invalidate the procedure calls like any other `fetch` calls.

According to the [SvelteKit docs](https://kit.svelte.dev/docs/load#invalidation) the appropriate `depends` should be called automatically when using `load`-`fetch`, however in my tests I could not get SvelteKit's invalidation to work without manually adding the procedure URLs with `depends`.
Does anybody know why this might be the case?

**Comment by Radiergummi** [comment]
Hi @MoritzKronberger, did you find a good answer for this? Trying to do the same thing. It feels wrong to manually create strings for the procedure URLs..
